test/rst_doc.md

```diff
@@ -1,3 +1,4 @@
+```{eval-rst}
```

Comment:
Is this to simplify build configuration? Having this markdown/rst hybrid format be the default isn't ideal to me. It makes a less-frequently used package a hard dependency and requirement for long term maintenance.

Reply:
Sphinx requires only one file format. To allow to test everything at once, we need md files + `{eval-rst}`.

- MyST parser needs to be tested to assert the parsing is done solely by Sphinx and the phpdomain does not operate on raw rst. This is related with #55 (I plan to fix it in another PR).

- `{eval-rst}` seems to work perfectly as native rst, Sphinx simply does not read rst file directly, but instead the wrapped string. I did not experience any difference between native rst file and rst from evaluated string.

MyST parser is very popular and the dev/test dependency should not be a problem.

Reply:
> MyST parser is very popular and the dev/test dependency should not be a problem.

Fair enough, this is the first time I've encountered it.